src/core/openthread-core-default-config.h

```diff
@@ -195,6 +195,36 @@
 #define OPENTHREAD_CONFIG_MAX_STATECHANGE_HANDLERS          1
 #endif  // OPENTHREAD_CONFIG_MAX_STATECHANGE_HANDLERS
 
+/**
+ * @def OPENTHREAD_CONFIG_COAP_ACK_TIMEOUT
+ *
+ * Minimum spacing before first retransmission when ACK is not received (RFC7252 default value is 2).
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_COAP_ACK_TIMEOUT
+#define OPENTHREAD_CONFIG_COAP_ACK_TIMEOUT                  2
+#endif  // OPENTHREAD_CONFIG_COAP_ACK_TIMEOUT
+
+/**
+ * @def OPENTHREAD_CONFIG_COAP_ACK_RANDOM_FACTOR
+ *
+ * Value used to calculate maximum spacing before first retransmission when ACK is not received
+ * (RFC7252 default value is 1.5, must not be decreased below 1).
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_COAP_ACK_RANDOM_FACTOR
+#define OPENTHREAD_CONFIG_COAP_ACK_RANDOM_FACTOR            1
```

Comment:
I realize this is an integer here, but we should be cognizant of this statement from RFC 7252:

>   ACK_RANDOM_FACTOR MUST NOT be decreased below 1.0, and it SHOULD have
>   a value that is sufficiently different from 1.0 to provide some
>   protection from synchronization effects.

Maybe define something like numerator/denominator values?